client/my-sites/upgrades/navigation.jsx

```diff
@@ -123,14 +117,12 @@ var UpgradesNavigation = React.createClass( {
 		var items;
 
 		if ( this.props.selectedSite.jetpack ) {
-			items = [ 'Plans', 'My Upgrades' ];
-		} else if ( config.isEnabled( 'upgrades/domain-management/list' ) ) {
-			items = [ 'Plans', 'Domains', 'Email', 'My Upgrades' ];
+			items = [ 'Plans', 'My Purchases' ];
```

Comment:
Well this is just an array key, it won't be seen by the user, and it will be removed in #1028.